scripts/Tools/case.test_build

```diff
@@ -17,6 +17,7 @@ from CIME.SystemTests.ncr import NCR
 from CIME.SystemTests.noc import NOC
 from CIME.SystemTests.pea import PEA
```

Comment:
I didn't replace case.build with case.test_build in the repo, only in the case.   We may be able to merge them as a second step, let me think about it a bit.